### DISCUSSION TITLE
firefox: URL argument opens as a blank tab

### DISCUSSION THREAD

**Discussion by rieje** [netblue30/firejail#5515] [Q&A] [answered]
I have an unchanged script that has always worked until at least the past week. When I pass url as an argument, it no longer opens the url, only a blank tab.

`bash -x <script>` shows the resulting command being run:

`firejail --seccomp --private /usr/bin/firefox --no-remote --profile /tmp/rieje/firefox-tmp-profiles/firefox-profile.5PorvOk5id --private 'https://vid.puffyan.us/watch?v=8jfXL0ePV7U'`

Any ideas? There was no recent Firejail update nor a Firefox update to indicate it might have been the culprit. `/usr/bin/firefox <url>` continues to work as expected (no Firejail).

**Comment by kmk3** [comment]
@rieje on [Dec 8](https://github.com/netblue30/firejail/discussions/5515#discussion-4640169):

> I have an unchanged script that has always worked until at least the past
> week. When I pass url as an argument, it no longer opens the url, only a
> blank tab.
> 
> `bash -x <script>` shows the resulting command being run:
> 
> `firejail --seccomp --private /usr/bin/firefox --no-remote --profile /tmp/rieje/firefox-tmp-profiles/firefox-profile.5PorvOk5id --private 'https://vid.puffyan.us/watch?v=8jfXL0ePV7U'`
> 
> Any ideas? There was no recent Firejail update nor a Firefox update to
> indicate it might have been the culprit. `/usr/bin/firefox <url>` continues
> to work as expected (no Firejail).

Maybe firejail is interpreting
`--profile /tmp/rieje/firefox-tmp-profiles/firefox-profile.5PorvOk5id`.

What is the command-line that the child process gets?

For example, see with `pstree -a "$USER"`.

Does it work with the following?

```sh
firejail --seccomp --private -- /usr/bin/firefox --no-remote \
  --profile /tmp/rieje/firefox-tmp-profiles/firefox-profile.5PorvOk5id \
  --private 'https://vid.puffyan.us/watch?v=8jfXL0ePV7U'
```

**Reply by rusty-snake** [reply]
No, firejail will stop at the first positional argument.

**Comment by rieje** [comment, marked as answer]
I no longer have this problem anymore, very strange. I still haven't done any modifications to firejail/firefox or the script that runs it.
